e2e_tests/conftest.py

```diff
@@ -78,15 +78,18 @@ def route_intercept(route):
 
 @pytest.fixture
 def create_user_page(
-    page, browser_name
+    page,
 ) -> (
     Page
 ):  # FIXME: browser_name specified until https://github.com/microsoft/playwright-pytest/issues/172 fixed
     # so that multiple browser flags in cli are honoured
     def _create_user_page(username, password) -> Page:
         page.goto("/sign-in")
-        page.get_by_label("Email address").fill(username)
-        page.get_by_label("Password").fill(password)
+        if page.locator("label:has-text('Email')").count() > 0:
```

Comment:
this is to account for the different styling we have locally compared to deployments of keycloak. Maybe we should align those at some point

Reply:
https://national-archives.atlassian.net/browse/AYR-1683